recipes/face_recognition_models/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "face_recognition_models" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: b79bd200a88c87c9a9d446c990ae71c5a626d1f3730174e6d570157ff1d896cf
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - setuptools
+
+test:
+  imports:
+    - face_recognition_models
+
+about:
+  home: "https://github.com/ageitgey/face_recognition_models"
+  license: CC0-1.0
+  license_family: CC
+  license_file: LICENSE
+  summary: "Models used by the face_recognition package."
+  doc_url: https://pypi.org/project/face_recognition_models/
+  dev_url: https://github.com/ageitgey/face_recognition_models
+
+extra:
+  recipe-maintainers:
+    - ndmaxar
+    - rluria14
+    - oblute
```

Comment:
@rluria14 and @oblute Are you happy to be listed as maintainers?

Reply:
@chrisburr yes, happy to be a maintainer

Reply:
@chrisburr same here.